test-plans/e2e-01-auth-and-roles.md

```
@@ -50,26 +52,25 @@ agent-browser screenshot
 
 **Check:** After clicking the theme toggle, the theme changes from dark to light.
 
 **Expected result:**
-- The return value from `getAttribute('data-theme')` is `"light"`
-- The page background changes to light colors (white/`#ffffff`)
-- The sun icon is now hidden and the moon icon is visible
+- The return value is `false` (dark class removed --- now light mode)
+- The page background changes to light colors
 
 **Then toggle back:**
 ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)